app/packs/stylesheets/dropzone.scss

```diff
@@ -166,14 +162,14 @@
   -o-animation: pulse 6s ease infinite;
   animation: pulse 6s ease infinite; }
 .dz-preview.dz-error .dz-error-message {
-  display: block; }
-.dz-preview.dz-error:hover .dz-error-message {
```

Comment:
This is about errors, so can this be reverted?

Reply:
it can be, but what it does is make the error message itself hovertext over the fileupload error-mark.   So I don't think we'll want it in any case.